internal/pkg/term/progress/ecs.go

```diff
@@ -180,6 +200,98 @@ func (c *rollingUpdateComponent) renderAlarms(out io.Writer) (numLines int, err
 	return renderComponents(out, components)
 }
 
+type stoppedTasksInfo struct {
+	ids              []string
+	latestStoppingAt time.Time
+}
+
+func (c *rollingUpdateComponent) renderStoppedTasks(out io.Writer) (numLines int, err error) {
+	if len(c.stoppedTasks) == 0 {
+		return 0, nil
+	}
+	header := []string{"TaskId", "CurrentStatus", "DesiredStatus"}
+	var rows [][]string
+	title := fmt.Sprintf("Latest %d %s stopped reason", len(c.stoppedTasks), english.PluralWord(len(c.stoppedTasks), "task", "tasks"))
+	title = fmt.Sprintf("%s%s", color.DullRed.Sprintf("✘ "), color.Faint.Sprintf(title))
+	childComponents := []Renderer{
+		&singleLineComponent{}, // Add an empty line before rendering task stopped events.
+		&singleLineComponent{
+			Text:    title,
+			Padding: c.padding,
+		},
+	}
+
+	taskInfoMap := make(map[string]stoppedTasksInfo, len(c.stoppedTasks))
```

Comment:
```suggestion
	stopReason2Tasks := make(map[string]stoppedTasksInfo, len(c.stoppedTasks))
```

In addition - this is not a request, just a random thought - perhaps we could do `map[string]*stoppedTasksInfo` to reduce the amount of swapping?